folium/plugins/__init__.py

```diff
@@ -29,6 +29,7 @@
 from folium.plugins.timestamped_geo_json import TimestampedGeoJson
 from folium.plugins.timestamped_wmstilelayer import TimestampedWmsTileLayers
 from folium.plugins.search import Search
+from folium.plugins import pattern
```

Comment:
The API currently is to expect the plugin classes to be listed here and not another module, so let's change this to

```python
from folium.plugins.pattern import CirclePattern, StripePattern
```